### ISSUE TITLE
More issues with too low contrast

### ISSUE THREAD

**Issue by torgeirl** [devilry/devilry-django#1183]
#1109 addressed a few issues with too low contrast, but there are more issues that have to be solved in order to pass success Criteria 1.4.3 in WCAG 2:

* `Search ...` in search field at i.e. `/devilry_student/` (handled by the `cradmin-legacy-clearable-textinput` class)

* chevrons at i.e. `/devilry_student/` (handled by the `fa-angle-right devilry-cradmin-legacy-listbuilder-itemframe-goforward-icon` class)

* date and time text for event logs in student feedback feed (handled by the `devilry-group-feedbackfeed-event-message devilry-group-feedbackfeed-itemvalue`, `devilry-group-feedbackfeed-event-message__grade` and `timeline-datetime` classes)

* icons of the comment editor in student feedback feed (handled by the `controls` class I believe)

* white on turquoise used for warnings displayed in student feedback feed for i.e. hard deadlines (handled by the `alert`, `alert-info` and `devilry-feedbackfeed-hard-deadline-info-box` classes)

* green on white used for the project group details link in the student version of the student feedback feed (handled by the `devilry-feedbackfeed-student-projectgroup-detauils-link` class)

**Comment by StianJul** [comment]
The scope of this task should be a milestone where the focus is the WCAG-related issues.
